simulator/infra/instrcache/LRUCache.h

```diff
@@ -73,31 +73,26 @@ class LRUCache
         void erase( const Key& key)
         {
             auto data_it = data.find( key);
-            auto lru_it  = lru_hash.find( key);
-            assert ( ( data_it == data.end()) == ( lru_it == lru_hash.end()));
+            assert ( data_it == data.end());
```

Comment:
Not sure that assertion make sense now. `erase` handles both cases: when element is found and when element is not found (do nothing in that case). Assertion checked that element is found simultaneously in data storage and LRU array, or not found simultaneously as well.